components/oauth/ApplicationApproveScreen.js

```diff
@@ -83,14 +83,16 @@ export const ApplicationApproveScreen = ({ application, redirectUri, autoApprove
     const body = await response.json();
     if (response.ok) {
       setRedirecting(true);
-      return router.push(body['redirect_uri']);
+      setTimeout(() => {
+        return router.push(body['redirect_uri']);
+      }, 1500);
```

Comment:
I would personally like to keep this value low, in live environments the network delays are also to take into account. The issue mentioned 1sec, which might be a good start:

```suggestion
      }, 1000);
```

Reply:
Awesome, thanks @Betree for the quick review!